pages/common/yank.md

```diff
@@ -0,0 +1,23 @@
+# yank
+
+> Read input from stdin and display a selection interface that allows a field to be selected and copied to the clipboard.
+
+- Yank an environment variable key or value:
+
+`env | yank -d =`
+
+- Yank a field from a CSV file:
+
+`yank -d \", <file.csv`
+
+- Yank a whole line using the -l option:
+
+`make 2>&1 | yank -l`
+
+- If stdout is not a terminal the selected field will be written to stdout and exit without invoking the yank command. Kill the selected PID:
```

Comment:
This description is a bit too long. Can we shorten this ?

Reply:
It comes from the documentation of the tool and I don't really see how to shorten it. But we can delete this specific example if you think it's not ok.

Reply:
I see that you have just lifted the descriptions verbatim from the documentation. We usually explain the descriptions in a language simple enough for beginners to understand. How about - `Select a field and pipe to stdout without copying to clipboard` ?